keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -65,7 +65,7 @@
             }
           },
           "default": {
-            "description": "Default response. It will be deserialized as per the error definition specified in the schema. An exception will be thrown.",
+            "description": "Default response. It will be deserialized per the error definition specified in the schema. An exception will be thrown.",
```

Comment:
I can see what @johanste means; this is supposed to be HTTP/REST oriented, yet the text is leaning very libraryish. That said, in the specs I've worked in response default was already present/generated. After scanning a few more of them, it seems "CDN error response describing why the operation failed." is commonly used. I would have to defer to @amarzavery on this one.